pkg/resourcequotas/resourcequotas.go

```diff
@@ -153,10 +153,10 @@ func (builder *Builder) Update(force bool) (*Builder, error) {
 				glog.V(100).Infof(msg.FailToUpdateError("resource quota",
 					builder.Definition.Name, builder.Definition.Namespace))
 
-				return nil, err
+				return builder.Create()
 			}
 
-			return builder.Create()
+			return nil, err
```

Comment:
@sebrandon1  I think this was fine before, I usually add an extra `return nil, err` outside of the `if force` but inside the `if err != nil` like [here in kac.go](https://github.com/openshift-kni/eco-goinfra/blob/main/pkg/ocm/kac.go#L222-L228).

The issue with this change is it puts the `builder.Create()` inside the inner `if err != nil` which means it would only recreate the resource if the delete also failed, not when the delete is successful